### PR TITLE
Add css exports to package.json

### PR DESCRIPTION
**Pull request by gmiller-in-situ**
My application was having trouble importing the ng-select themes into less after an update to Angular 14. We found that the files needed to be exported in the package.json so they could be found by the builder. This is backpacking off a similar commit recently that exported the .scss files, but missed exporting the .css files.

### CODE REVIEW

**Review by antonymarion — APPROVED** [review]

**Comment by antonymarion** [comment]
@varnastadeus : ok for you ?

**Review by pavankjadda — APPROVED** [review]

**Comment by pavankjadda** [comment]
The library fails in Angular v15. This PR fixes the export bug. 
```
Error: Package path ./themes/material.theme.css is not exported from package /Users/jaddap2/WebstormProjects/PRES-UI/node_modules/@ng-select/ng-select (see exports field in /Users/jaddap2/WebstormProjects/PRES-UI/node_modules/@ng-select/ng-select/package.json)
```

@varnastadeus please approve this PR

**Comment by antonymarion** [comment]
> The library fails in Angular v15. This PR fixes the export bug.
> 
> ```
> Error: Package path ./themes/material.theme.css is not exported from package /Users/jaddap2/WebstormProjects/PRES-UI/node_modules/@ng-select/ng-select (see exports field in /Users/jaddap2/WebstormProjects/PRES-UI/node_modules/@ng-select/ng-select/package.json)
> ```
> 
> @varnastadeus please approve this PR

Yes, @varnastadeus I would really appreciate if you please do any of the following action :

- explain why you dont want to merge this tiny fix
- merge it either

We are today 7 Nov so 2 months after my question: `is it ok for you to merge this ?`

I am currently using this post install npm script to apply this change.. 

````
const path = require("path");
const replace = require('replace-in-file');
const pathToNgSelectPackageJSON = path.join(__dirname, 'node_modules/@ng-select/ng-select/package.json');
const options = {
  files: pathToNgSelectPackageJSON,
  from: `    "./package.json": {
      "default": "./package.json"
    },`,
  to: `    "./themes/default.theme.css": {
      "style": "./themes/default.theme.css"
    },
    "./themes/material.theme.css": {
      "style": "./themes/material.theme.css"
    },
    "./package.json": {
      "default": "./package.json"
    },`,
};

try {
  const results = replace.sync(options);
  console.log('patched @ng-select package JSON for css imports:', results);
}
catch (error) {
  console.error('Error occurred:', error);
}
````

**Comment by pavankjadda** [comment]
@varnastadeus I am not sure how much active you are in Angular community, if you are not I would suggest adding one more maintainer to the project. If you have someone in mind, please add them. If not, I am happy to do it. FYI I co-maintain [ngx-cookie-service](https://github.com/stevermeister/ngx-cookie-service)

**Review by varnastadeus — APPROVED** [review]

**Pull request merged**
